resources/views/auth/register.blade.php

```diff
@@ -55,10 +55,9 @@
                                 @endif
                             </div>
                         </div>
-
+                        <!--
```

Comment:
remove if you do not need

Reply:
Changed in line with the latest Wire Frame.